lib/action_view/component/validatable.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module ActionView
+  module Component # :nodoc:
+    module Validatable
+      extend ActiveSupport::Concern
+
+      included do
+        if ActionView::Component.extend_validations
+          include ActiveModel::Validations
+        end
```

Comment:
What is the importance of making this configurable? What's the cost of including this always even if someone is overriding `validate!`?

Reply:
I think it is not correct to load module that not used. It might not to has a big cost probably, but it is additional bytes/miliseconds that is unneccessary. People who prefer dry ecosystem and sequel make this choice because of performance as pointed by @rainerborene [here](https://github.com/github/actionview-component/pull/192#issuecomment-580766349) 

If we make optional validation mechanism, I think the next step could be to make `ActiveModel` optional at all